pkg/arvo/tests/sys/zuse/html.hoon

```diff
@@ -211,7 +210,6 @@
     ['[\E2\81\A0]' "n_value_U+2060_word_joined_array"]
     ['{\E2\81\A0}' "n_value_U+2060_word_joined_object"]
     ['\\u0020"asd"' "n_value_UTF16_outside_string"]
-    ['\EF\BB\BF' "n_value_UTF8_BOM_no_data"]
```

Comment:
Any reason not to just keep these in?

Reply:
I found that there is no test in the JSON test suite for a valid BOM, and that all of the BOM tests were getting caught by tests for text outside of double-quotes, so I removed them to keep the test suite concise while still checking all cases.

I could add the BOM tests back in, if the consensus is that they cover a valid test case.

Reply:
@ohAitch @joemfb 
Any new opinions regarding these particular test cases?